src/tablet/tablet_impl.cc

```diff
@@ -2944,6 +2944,38 @@ int TabletImpl::LoadTableInternal(uint32_t tid, uint32_t pid, std::shared_ptr<::
         ::openmldb::storage::Binlog binlog(replicator->GetLogPart(), binlog_path);
         if (snapshot->Recover(table, snapshot_offset) &&
             binlog.RecoverFromBinlog(table, snapshot_offset, latest_offset)) {
+            // recover aggregator if exists
+            std::string aggr_path = GetDBPath(db_root_path, tid, pid) + "/aggr_info.txt";
+            if (::openmldb::base::IsExists(aggr_path)) {
+                int fd = open(aggr_path.c_str(), O_RDONLY);
+                ::openmldb::api::CreateAggregatorRequest request;
+                if (fd < 0) {
+                    PDLOG(ERROR, "[%s] is not exist", aggr_path.c_str());
+                } else {
+                    google::protobuf::io::FileInputStream fileInput(fd);
+                    fileInput.SetCloseOnDelete(true);
+                    if (!google::protobuf::TextFormat::Parse(&fileInput, &request)) {
+                        PDLOG(WARNING, "parse create aggregator meta failed");
```

Comment:
need not to create if parse failed

Reply:
@dl239 If it fails, the else block will not be executed.